server/views/pages/optOut/optOutCheck.njk

```diff
@@ -19,11 +21,8 @@
             </p>
             <form method="POST">
                 <input type="hidden" name="_csrf" value="{{ csrfToken }}">
-                {% if hasError %}
-                    {% set errorMessage = 'Select a value' %}
-                {% endif %}
                 {{ govukRadios({
-                    idPrefix: "optOutRadio",
+                    idPrefix: "considerForHdc",
```

Comment:
this fixes the link so when you select the link in the error summary it puts the first option of the associated question in focus